tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -918,11 +920,63 @@ public function it_should_create_mime_types_for_allowed_sizes_only_via_global_va
 		$this->assertImageNotHasSizeSource( $attachment_id, 'not_allowed_size_200x150', 'image/webp' );
 	}
 
+	/**
+	 * Test image quality for image conversion.
+	 *
+	 * @test
+	 */
+	public function test_set_quality_with_image_conversion() {
+		global $wp_version;
+		// Set conversions for uploaded images.
+		add_filter( 'image_editor_output_format', array( $this, 'image_editor_output_formats' ) );
+		$expected_webp = 82;
+		if ( version_compare( '6.2', $wp_version ) ) {
+			$expected_webp = 86;
+		}
```

Comment:
See above, this is not right: The WebP quality after this fix should _always_ be 82. The only difference is in how we fix this in the filter, depending on whether it's WP <6.1 or WP >=6.1.

Reply:
Updated the logic to reflect the changes in the test case. Please review once.